pydantic/functional_validators.py

```diff
@@ -388,16 +412,24 @@ def __call__(
 
 
 class ModelAfterValidatorWithoutInfo(Protocol):
+    """A @model_validator decorated function signature.
+    This is used when `mode='after'` and the function does not have info argument.
+    """
+
     @staticmethod
-    def __call__(
+    def __call__(  # noqa: D102
         self: _ModelType,  # type: ignore
     ) -> _ModelType:
         ...
 
 
 class ModelAfterValidator(Protocol):
+    """A @model_validator decorated function signature.
+    This is used when `mode='after'`.
+    """
```

Comment:
We have 120 line length 👀 
```suggestion
    """A @model_validator decorated function signature. This is used when `mode='after'`."""
```